bond.go

```diff
@@ -28,6 +30,10 @@ type HasAfterCreate interface {
 	AfterCreate()
 }
 
+type hasAfterCreateTx interface {
```

Comment:
My proposal is to have an `AfterCreate()` method that accepts a `bond.Session` pointer as parameter, this `bond.Session` can be used instead of a global DB, and will pass the transaction context when the `AfterCreate` is called in such context.

I made the argument optional just to show the differences, the original `AfterCreate()` method will not be affected by this new interface.